tsconfig.base.json

```diff
@@ -7,7 +7,8 @@
     "forceConsistentCasingInFileNames": true,
     "declaration": true,
     "declarationMap": false,
-    "sourceMap": true
+    "sourceMap": true,
+    "resolveJsonModule": true
```

Comment:
Our public interface allows people to import `@yext/answers-core/lib/commonjs` or `@yext/answers-core/lib/esm`. When we import the package.json file from the src files, the output files in the lib directories are restructured since the src files need to be able to import the package.json file. This means that the imports listed above will break since there is no longer an index.js file in those folders. The package.json files will be read instead, and the `main` paths in the package.json files of those folders will be invalid.

One solution is to update the outDir of the tsconfig.esm.json and the tsconfig.commonjs.json files to be dist/esm and dist/commonjs. Then we can check-in the lib folder into the repo and add small package.json files which point to the corresponding index and bundle.d.ts files in dist. You can see an example of this with legacy/package.json, however we should use the `main` and `module` keys instead of `browser` to point to the code.